### PR TITLE
update server instructions for self-signup and pullrequest version 3

### PR DESCRIPTION
**Pull request by phritz**

### CODE REVIEW

File: SERVER_SETUP.md [hunk 1]
```diff
@@ -46,17 +46,32 @@ The key/value pairs you return are up to you — Replicache just makes sure they
 
 See [client-view-schema.jsonc](./client-view-schema.jsonc) for the schema of the Client View response.
 
-#### Authentication
+#### Client View Authentication
 
 Most applications return a Client View that is specific to the calling user. Replicache supports sending user credentials through the standard `Authorization` HTTP header. If authorization fails, the Client View should return HTTP 401 (Unauthorized). The Replicache client will prompt user code to reauthenticate in that case.
 
+Note that if your Client View does not authenticate users via the `Authorization` header, there is nothing preventing users from fetching each others' data.
```
Comment (aboodman) [inline SERVER_SETUP.md]: Is there something Replicache specific about this? It just seems like something general about web apis...

File: SERVER_SETUP.md [hunk 2]
```diff
@@ -46,17 +46,32 @@ The key/value pairs you return are up to you — Replicache just makes sure they
...
 
 All responses other than HTTP 200 with a valid JSON Client View and HTTP 401 are treated as errors by the Diff Server. The Client View response is ignored and the app is sent the last known state instead.
 
 ### Step 2: Test Downstream Sync
 
-Download diff-server:
+The request to your Client View goes through the Diff Server, which turns the full Client View your service returns into a smaller delta to be applied the state if any that your client has. If your service is publicly accessible, you can use the Diff Server service we provide. If not, for example your Client View is running on `localhost` or at some other not-public address, you can run the Diff Server yourself. 
+
+Note that running the Diff Server yourself is convenient for evaluating Replicache, but in order to launch to users "for real"
+your Client View will need to be publicly accessible, and you will need to create a Replicache account as outlined in Step 2a.
```
Comment (aboodman) [inline SERVER_SETUP.md]: "Step 2a" doesn't appear anywhere below.

**Review by aboodman — APPROVED** [review]

**Comment by aboodman** [comment]
Whoops missed "do not review".

**Comment by phritz** [comment]
I was editing in the web interface because it's nice to see the changes
then realized how much of a pain in the ass it was going to be if someone
else was also making changes so saved the changes to a branch to work
locally. It's def WIP.

**Comment by arv** [comment]
If you use vscode you can use markdown preview.

**Comment by phritz** [comment]
NICE, didn't know that was a feature!

**Comment by aboodman** [comment]
In general in vs code:

cmd+shift+p, <the thing you want to do>

File: SERVER_SETUP.md (hunk 1 above)
Comment (phritz) [reply SERVER_SETUP.md]: You greatly overestimate the capacity for attention to detail in the general population.

**Comment by phritz** [comment]
ok @aboodman this is ready for a look

**Pull request merged**